Procfile

```diff
@@ -1 +1 @@
-web: DEBUG=socket.io*,refocus-real-time* npm start
+web: npm start
```

Comment:
we can use `DEBUG=$DEBUG` then we have the ability to turn some debug on via env var

Reply:
That shouldn't be necessary, we can turn it on as-is. This is just to prevent it from always being on.

Reply:
What do you mean "turn it on as-is"? I want us to be able to turn on without deploying code, i.e. just by setting an env var.